packages/runtime-common/realm.ts

```diff
@@ -517,9 +532,9 @@ export class Realm {
     card.data.links = { self: url.href };
     return new Response(JSON.stringify(card, null, 2), {
```

Comment:
this is an example of a response that should not be instantiate. There should be no new Response, rather the response that is used should be the one that is passed in.